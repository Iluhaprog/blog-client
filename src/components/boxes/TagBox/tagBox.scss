@import '../../../vars.scss';

.tag {
	font-size: 13px;
	font-weight: 400;
	padding: 5px;
	margin: 0 5px 0 0;
	background-color: $text-color-6;
	border-radius: 5px;
	color: $text-color-1;

	&__violet {
		background-color: $nav-bg;
	}
}
