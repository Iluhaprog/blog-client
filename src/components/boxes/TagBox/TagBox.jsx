import React from 'react';
import './tagBox.scss';

export default props => (
    <li className='tag'>
        { props.title }
    </li>
)