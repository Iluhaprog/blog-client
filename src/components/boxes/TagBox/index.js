import TagBox from './TagBox';

export {
    TagBox,
};