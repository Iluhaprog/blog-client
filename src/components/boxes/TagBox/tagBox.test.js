import React from 'react';
import { render } from '@testing-library/react';
import TagBox from './TagBox';

test('Test view of tag box', () => {
    const { asFragment } = render(
        <TagBox title='Test title' />
    );
    expect(asFragment()).toMatchSnapshot();    
});