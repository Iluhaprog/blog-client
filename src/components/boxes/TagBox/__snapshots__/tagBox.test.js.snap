// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test view of tag box 1`] = `
<DocumentFragment>
  <li
    class="tag"
  >
    Test title
  </li>
</DocumentFragment>
`;
